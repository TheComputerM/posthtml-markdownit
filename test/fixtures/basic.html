<md>
    # Heading 1
    ---

    Paragraph with some text

    *Italic*
    __Bold__

    - List item 1
    - List item 2
    - List item 3
</md>