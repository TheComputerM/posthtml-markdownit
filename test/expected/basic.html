<div>
<h1>Heading 1</h1>
<hr>
<p>Paragraph with some text</p>
<p><em>Italic</em>
<strong>Bold</strong></p>
<ul>
<li>List item 1</li>
<li>List item 2</li>
<li>List item 3</li>
</ul>
</div>
